lib/marqeta/transaction.rb

```diff
@@ -3,6 +3,12 @@ class Transaction < ApiObject
     PENDING_STATE = 'PENDING'.freeze
     DECLINED_STATE = 'DECLINED'.freeze
 
+    DECLINED_BY_JIT_RESPONSE_CODE = '402'.freeze
+    TIMEOUT_RESPONSE_CODE = '97'.freeze
+    JIT_ERROR_RESPONSE_CODE = '96'.freeze
+    EXCEEDING_AMOUNT_LIMIT_RESPONSE_CODE = '1834'.freeze
+    EXCEEDING_COUNT_LIMIT_RESPONSE_CODE = '1817'.freeze
```

Comment:
From the docs, it looks like the last two here are "transaction response codes", but the first two are some other kind – I think it would be useful to distinguish these, e.g.:

```
module Marqueta::TransactionCodes
  EXCEEDING_AMOUNT_LIMIT = '1834'.freeze
  ...
end
```

```
module Marqeta::NetworkSomethingSomething
  TIMEOUT = '97'.freeze
  ...
end
```

Reply:
@andyw8 sounds good 👍